honeycomb-core/src/embed.rs

```diff
@@ -95,7 +96,7 @@ pub enum UnsewPolicy {
 }
 
 /// Type definition for 2D vertex representation.
-pub type Vertex2 = [f64; 2];
+pub type Vertex2<T> = Coords2<T>;
```

Comment:
Good to have a separate type. You can later add attribute like a name or different coordinates systems.